plugins/catalog-backend/src/processors/UrlReaderProcessor.ts

```diff
@@ -128,23 +132,13 @@ export class UrlReaderProcessor implements CatalogProcessor {
     location: string,
     etag?: string,
   ): Promise<{ response: { data: Buffer; url: string }[]; etag?: string }> {
-    // Does it contain globs? I.e. does it contain asterisks or question marks
-    // (no curly braces for now)
-
-    const { filepath } = parseGitUrl(location);
-    if (filepath?.match(/[*?]/)) {
-      const response = await this.options.reader.search(location, { etag });
-      const output = response.files.map(async file => ({
-        url: file.url,
-        data: await this.#limiter(file.content),
-      }));
-      return { response: await Promise.all(output), etag: response.etag };
-    }
+    const response = await this.options.reader.search(location, { etag });
+
+    const output = response.files.map(async file => ({
+      url: file.url,
+      data: await this.#limiter(file.content),
+    }));
```

Comment:
Let's move this new behavior behind a config flag so that people can opt-in, and then I think this is good to go 👍 

Let's also log a deprecation warning if the flag is not set to encourage people to move over

Reply:
Done!